src/index.js

```diff
@@ -0,0 +1,26 @@
+const url = "https://api.github.com/search/users?q=";
+const userMode = "%20in:login";
+let item = document.querySelector(".items");
+const button = document.querySelector(".search_button");
+button.addEventListener("click", displayUser)
+
+async function getResource() {
+  let nameMode = document.querySelector(".search_input").value;
+  if (nameMode) {
+    const res = await fetch(url + nameMode + userMode);
+    // if (!res.ok) {
+    //     throw new Error(`Could not fetch ${url}` +
+    //         `, rceived ${res.status}`)
+    // }
+    return await res.json();
```

Comment:
1. never leave commented code in PR
2. such error handling is a bit senseless since you throw another error and never handle it later
3. if you still want to handle error use:
```js
try {
  const res = await fetch(url + nameMode + userMode);
  return await res.json();
} catch(e) {
  throw new Error(`Could not fetch ${url} ${res.status}`);
}
```